### PR TITLE
Define who publishers are, orgs and individuals

### PR DESCRIPTION
**Pull request by jonathansick**
This text is meant to specify that not only can formally
organized blogs publishers to the Archive, but individuals can also use
the Archive as a place to have their content indexed.

### CODE REVIEW

**Comment by jonathansick** [comment]
Essentially in this section I'm saying that there will be AIAC sub-communities for 'organizations' and the content is directly mapped to the works of that organization's blog, for example.

There will also be AIAC sub-communities created by _each_ individual, and these types of repos will be catch-alls for the material that that individual publishes to the web.

What say you @kelle @owlice @augustfly @libcce ? Is this the model we're going for?

**Comment by owlice** [comment]
I suggest having this initially for "organizations", with a blog being recognized as an organization (so AstroBetter/Astrobites/etc. would fall under this), smoothing out the workflow, and then adding individuals. Not that I think individuals should not be included (they should be!) -- just aiming for a manageable build of the resource. But Alberto may have something to say on that score; I think we would want some type of oversight to ensure what is getting archived is ... well, reasonable stuff.

**Comment by kelle** [comment]
I'm a bit confused here...I don't think sub-communities are a thing in Zenodo. 

I think what you are proposing is a whole different option...and actually a viable alternative which we have not discussed:
- Have AstroBetter/AstroBites/WIA Blog make their _own_ communities in Zenodo and ask ADS to index them.
- Use the AIAC community just for _individual_ blogs.

But I think the whole bit about sub communities is probably overthinking a bit....let's just make one Zenodo Community to start with where all "blog" content goes. Once we hash out the first draft of the kind of content we accept, the ~3 moderators/editors can make decisions about acceptable content and suggest changes to the rest of the group about possible changes.

**Comment by owlice** [comment]
My understanding is that Lars is working on implementing sub communities, and I think that's what is making this whole thing possible -- manageable in Zenodo and for ADS, too, so ADS is ingesting essentially from one source instead of a lot of small sources. That way different sub communities can be created as new blogs come online, each would have its own curators, and they would all be operating under the larger umbrella. 

This would provide more stability over the long-term, I suspect, as blogs come and go; ADS would still have just the one entity to ingest and the workflow would have already been worked out, saving time/effort as new blogs/etc. want to participate.

**Comment by jonathansick** [comment]
@kelle Regarding your proposal, perhaps @aaccomazzi (+ @libcce and @augustfly) might want to confirm what arrangement is tenable for ADS.

The first option is the one I originally wrote up suggesting that both organizations and individuals obtain their own Zenodo sub-communities (assuming those exist in a reasonable time frame).

The other option is @kelle's suggestion a couple posts above of:

> - Have AstroBetter/AstroBites/WIA Blog make their own communities in Zenodo and ask ADS to index them.
> - Use the AIAC community just for individual blogs.

In other words, our job here will be to create a workflow for established blogs to get indexed by ADS, and let them essentially self-govern. Then AIAC itself only provides the governance and archiving for individual's works. I kind of like @kelle's idea, but I only fear is that it's creating more work for ADS. @aaccomazzi any comment?

**Comment by kelle** [comment]
If @owlice insider info is right and sub-communities is a thing which is coming soon, then I think that's better! But yes, @aaccomazzi opinion matters a lot here.

**Comment by owlice** [comment]
As does @larshankat’s!
